proxyd/config.go

```diff
@@ -151,6 +151,7 @@ type BackendGroupConfig struct {
 	Backends []string `toml:"backends"`
 
 	WeightedRouting bool `toml:"weighted_routing"`
+	DisableShuffle  bool `toml:"disable_shuffle"`
```

Comment:
Maybe using `shuffle` is a better code style but I think everything needs to be by default `false`?